cypress/e2e/dashboard.cy.ts

```diff
@@ -31,7 +30,7 @@ describe('Dashboard', () => {
 
         insight.addInsightToDashboard(dashboardName, { visitAfterAdding: true })
 
-        cy.get('.CardMeta h4').should('have.text', insightName)
```

Comment:
unless i'm being slow this is a really different assertion

before it was that the new insight is in the dashboard in a card

now it's that the dashboard name is in the page

so you're not asserting that the insight is in the page at all - which means we could break it and the test would still be green

Reply:
Yeah sorry I was working on this, not final. Pushed a fixed version not that I believe does what we want.